synapseclient/client.py

```diff
@@ -689,6 +701,7 @@ def _getWithEntityBundle(self, entityBundle, entity=None, **kwargs):
         submission = kwargs.pop('submission', None)
         followLink = kwargs.pop('followLink', False)
         path = kwargs.pop('path', None)
+        max_threads = kwargs.pop('maxThreads', None) or self._get_transfer_config_max_threads()
```

Comment:
kwargs was originally used to avoid accidentally setting the wrong value for optional parameters in in positional args

Since we moved away from python 2, we can stop using kwargs in favor of Keyword-Only Arguments.

https://www.python.org/dev/peps/pep-3102/